bin/bovine.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import os
+import yaml
+##########################
+# Read the Static Directory
+##########################
+
+
+class Inventory:
+  def __init__(self, root_directory):
+    # get the files in the root_directory
+    # self.
+    self.root_directory = root_directory
+    # import the yaml from the files that we found above
+  def list_files(self, root_directory):
+    directory_files = os.walk(root_directory)
+    filenames = []
+    for root, dirs, files in directory_files:
+      for name in files:
+        filenames.append(os.path.join(root, name))
+    return filenames
+    # print(filenames)
+
+
+  def import_yaml(self, filenames):
```

Comment:
you shouldn't have to pass the filenames var explicitly here, if it's properly assigned to self in the list_files method above.